packages/vats/src/proposals/network-proposal.js

```diff
@@ -0,0 +1,162 @@
+// @ts-check
```

Comment:
automated tests for this proposal would be great, but not likely cost-effective.

I saw this run on @warner 's desk. Here's hoping someone finds a moment to reproduce his results. But I don't think it's critical.

Reply:
FWIW the tests we ran were:
* use cosmic-swingset's `make scenario2-run-chain` and `make scenario2-run-client`
* run `agoric open --repl`, open a browser to the URL it prints
* in the repl, do and observe:
```
command[11]    E(E.get(home.ibcport)[0]).getLocalAddress()
history[11]    "/ibc-port/port-1"
```